website/src/mode.tsx

```diff
@@ -123,38 +134,38 @@ export const ExperiencePicker = (p: { onSelect: (mode: ViewMode) => void }) => {
 export const PhoneticsPicker = (p: {
   onSelect: (phonetics: PhoneticRepresentation) => void
 }) => {
-  const radio = useRadioState({
-    state: selectedPhonetics(),
-  })
+  const [value, setValue] = useState(
+    selectedPhonetics() as PhoneticRepresentation
```

Comment:
Go ahead and move the `as ...` to the `selectedPhonetics` function so that you don't have to repeat it at every call site.